test/enum.js

```diff
@@ -55,14 +55,16 @@ api.metatests.test('Enum string keys', (test) => {
   test.strictSame(Month.has('May'), true);
   test.strictSame(may.value, '4');
 
+  test.strictSame(may, Month.from('May'));
+
   test.strictSame(+may, 4);
-  test.strictSame(may + '', 'May');
```

Comment:
I think it will be good to convert to values, not keys

Reply:
That's what I couldn't understand in the previous implementation. It converted toPrimitive based on type which imo should never be the case as it basically says that we cannot know what's the correct primitive value for this enum because it depends on the type. I think enum should always have it's own (internal, number) value as toPrimitive and offer actual key-value separately. That's why I added keyValue property to the enum.
On the other hand I think that it will be much clearer if user had to excplicitly say what is needed (key or value). This `may.keyValue` imo looks much better than `may + ''` that assumes specific toPrimitive conversion.
Also is there any use case where it's important and convenient to have toPrimitive convert to values (or even keys)? Maybe it's worth to just remove it from the enum completely?

Reply:
Ok, but `keyValue` is not clear naming. Maybe we can name it `.key`, so we will have `.key` and `.value`?